lib/pengine/group.c

```diff
@@ -269,10 +269,12 @@ pe__group_default(pcmk__output_t *out, va_list args)
 
     int rc = pcmk_rc_no_output;
 
-
     gboolean parent_passes = pcmk__str_in_list(only_rsc, rsc_printable_id(rsc), pcmk__str_none) ||
                              (strstr(rsc->id, ":") != NULL && pcmk__str_in_list(only_rsc, rsc->id, pcmk__str_none));
 
+    gboolean active = rsc->fns->active(rsc, TRUE);
+    gboolean partially_active = rsc->fns->active(rsc, FALSE);
```

Comment:
I hate bringing this up this late, but since we're staring down a release ... what do you think of the suggestion on the original PR to use e.g. "(2 members inactive)" instead of "(partially active)"?

The implementation wouldn't be difficult, we'd just have to do the same thing as group_active() with a count rather than boolean instead of the above active() calls.

Reply:
Updated, with the catch that it only displays "%s members inactive" if you are not displaying inactive resources.  If you are also displaying inactive resources, you probably don't need the additional information to know that some resources are not being shown.  With this, there's really only a couple spots in the test output that change.